client/src/main/java/org/schemarepo/client/Avro1124RESTRepositoryClient.java

```diff
@@ -93,7 +93,7 @@ public Subject lookup(String subject) {
   public Iterable<Subject> subjects() {
     ArrayList<Subject> subjectList = new ArrayList<Subject>();
     try {
-      String subjects = webResource.get(String.class);
+      String subjects = webResource.accept(MediaType.TEXT_PLAIN_TYPE).get(String.class);
```

Comment:
This worries me a little bit...

The intent of the ```Avro1124RESTRepositoryClient``` class is to have a test which emulates, as much as humanly possible, the behavior of the original client implementation in the AVRO-1124 ticket, so that folks using that code can upgrade to the schema-repo as seamlessly as possible.

Adding this accept clause seems to imply that the default (or unspecified) media type sent by the Jersey client implementation would break against the new schema repo server code. If this is false, then can we just remove the ```accept(MediaType.TEXT_PLAIN_TYPE)``` statement? If this is true, then does that mean this PR introduces server-side changes that are incompatible with the old (AVRO-1124) clients, and if so, is there anything we can do to prevent or mitigate that?